docs/_layouts/default.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
-<html lang="{{ site.lang | default: "en-US" }}">
+<html lang="en-US">
   <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8" />
```

Comment:
> do not close void elements, i.e. write <br>, not <br />

https://google.github.io/styleguide/htmlcssguide.html

Meta is a void element. https://www.w3.org/TR/2011/WD-html-markup-20110405/syntax.html#syntax-elements